pages/posts/w3c-wot-case-study.md

```diff
@@ -38,17 +38,19 @@ of physical entities called [Thing Descriptions][thing-description], and to
 model and describe data sent by Internet of Things consumers and producers in a
 multi-protocol manner.
 
-We are on-board with JSON Schema since Draft 4, and even the first draft
-versions were quite enough to model data models of Internet of Things devices.
-Currently, we are on Draft 7 and expect to move to 2020-12 or a newer one when
-starting our new charter in February 2023.
-
-We observe JSON Schema becoming more and more stable over the years and
-Postman's support to the community gives us even more confidence on it. JSON
-Schema is delivering exciting features, and more importantly these features are
-becoming more and more consistent across implementations.  For example,
-vocabularies are very promising and we plan to explore them further in
-conjunction with Semantic Web technologies.
+The W3C Web of Things are on-board with JSON Schema since Draft 4, and even the
+first draft versions were quite enough to model data models of Internet of
+Things devices.  "Currently, we are on Draft 7 and expect to move to 2020-12 or
+a newer one when starting our new charter in February 2023." - Ege Korkan
+
+"We observe JSON Schema becoming more and more stable over the years and
+Postman's support to the community gives us even more confidence on it." - Ege
+Korkan 
+
+JSON Schema is delivering exciting features, and more importantly these
+features are becoming more and more consistent across implementations.  For
+example, "vocabularies are very promising and we plan to explore them further
+in conjunction with Semantic Web technologies." - Ege Korkan
 
```

Comment:
I suggest you remove "- Ege Korkan", and instead write: For example, Ege Korkan added, "vocab[...]"